travis/kyaml-pre-commit.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
-set -e
-
-cd kyaml
-make all
+# Copyright 2019 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
 
-cd ../cmd/config
-make all
+set -e
```

Comment:
would be clearer if .travis.yml 'called' make targets directly, rather than bash scripts that call make (which then calls other scripts).  make has issues, but at least has a well-understood notion of dependence.